src/omv/ports/rp2/cambus.c

```diff
@@ -66,7 +66,7 @@ int cambus_deinit(cambus_t *bus)
 
 int cambus_scan(cambus_t *bus)
 {
-    for (uint8_t addr=0x09, rxdata; addr<=0x77; addr++) {
+    for (uint8_t addr=0x20, rxdata; addr<=0x48; addr++) {
```

Comment:
@kwagyeman Should probably do the same for STM32. There's no need to scan lower and higher addresses.